src/com/company/Team.java

```diff
@@ -0,0 +1,80 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Team {
+    private int teamId;
+    private String name;
+    private int noOfPlayers;
```

Comment:
no need of maintaining noOfPlayers variable. member.size() gives the same result. 
maintain a constant :
public static final int NUMBER_OF_PLAYERS_IN_A_TEAM = 11
and use this variable everywhere.

Reply:
Seems good. Done